src/config/index.js

```diff
@@ -21,7 +21,7 @@ const Config = {
   DEFAULT_GAS_PRICE: 0.0000004,
   CREATE_CORACLE_GAS_LIMIT: 3500000,
   CREATE_DORACLE_GAS_LIMIT: 1500000,
-  UNLOCK_SECONDS: 86400,
+  UNLOCK_SECONDS: 604800,
```

Comment:
how we define this?

Reply:
It's self defined. I changed it from 1 day to 1 week. This is just to be sure that when the user starts the dapp it wont run out of time.